app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -445,14 +447,41 @@ private void initViews() {
     }
 
     /**
-     * Creates bottom sheet behaviours from bottom sheets, sets initial states and visibility
+     * a) Creates bottom sheet behaviours from bottom sheets, sets initial states and visibility
+     * b) Gets the touch event on the map to perform following actions:
+     *      collapse fab if fab is open.
+     *      if bottom sheet details are expanded .
```

Comment:
Did you mean to write an "if ... then ..." sentence here too?
Thanks!

Reply:
Yeah,
Fixed those